security/pkg/nodeagent/cache/monitoring.go

```diff
@@ -44,6 +51,12 @@ var (
 	numFileSecretFailures = monitoring.NewSum(
 		"num_file_secret_failures_total",
 		"Number of times secret generation failed for files")
+
+	certExpiryMinutes = monitoring.NewGauge(
+		"cert_expiry_minutes",
```

Comment:
seconds may be more canonical: https://prometheus.io/docs/practices/naming/#base-units